TypeCobol.Test/Parser/Programs/TypeCobol/QualifiedVarSubsriptAccess.rdz.tcbl

```diff
@@ -0,0 +1,15 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 StringT.
+         02 DataT      Pic X(10000).
+         02 LengthT    Pic 99999.
+       PROCEDURE DIVISION.
+      *Ok
+             Move Low-Values       To StringT (1 : LengthT)
+      *Ko syntax error
```

Comment:
Comment is wrong, this syntax is OK in TC.
Can you do the same test in Cobol to check that there is an error ?

Reply:
The test using -cob option is a CLI test called failcoboloptionparse
Change comment in 3174661